dome9/data_source_dome9_admission_control_policy_test.go

```diff
@@ -0,0 +1,65 @@
+package dome9
+
+import (
+	"github.com/terraform-providers/terraform-provider-dome9/dome9/common/testing/variable"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+
+	"github.com/terraform-providers/terraform-provider-dome9/dome9/common/resourcetype"
+	"github.com/terraform-providers/terraform-provider-dome9/dome9/common/testing/method"
+)
+
+func TestAccDataSourceAdmissionControlPolicyBasic(t *testing.T) {
+	// Generate All Required Random Names for Testing
+	policyTypeAndName, policyDataSourceTypeAndName, policyGeneratedName := method.GenerateRandomSourcesTypeAndName(resourcetype.AdmissionControlPolicy)
+	// t.Log("policyTypeAndName: " + policyTypeAndName)
```

Comment:
please remove the commented lines

Reply:
i just include the debugs if someone want to debug the tests and to make the test outputs more clear when runs the whole tests (don't want to print too much info), and commented them if someone want to debug in future, it was helpful , but in case this is not recommended , i will remove them

Reply:
all commented lines had been removed